Aquarius/Src/Aquarius/Renderer/SpriteSheet.cpp

```diff
@@ -0,0 +1,35 @@
+#include "SpriteSheet.h"
+
+#include "Aquarius/Renderer/QuadTexCoords.h"
+
+
+namespace Aquarius {
+
+	SpriteSheet::SpriteSheet(Texture* texture, uint32_t spriteWidth, uint32_t spriteHeight)
+	{
+		CreateSpriteSheet(texture, spriteWidth, spriteHeight);
+	}
+
+	void SpriteSheet::CreateSpriteSheet(Texture* texture, uint32_t spriteWidth, uint32_t spriteHeight)
+	{
+		m_Texture = texture;
+		m_TotalWidth = texture->getWidth();
+		m_TotalHeight = texture->getHeight();
+		m_SpriteWidth = spriteWidth;
+		m_SpriteHeight = spriteHeight;
+	}
+
+	QuadTexCoords SpriteSheet::getSpriteCoords(uint32_t i, uint32_t j) const
+	{
+		// Bottom Left Zero Indexed
```

Comment:
So the bottom left of the sprite sheet is (0,0)?

Reply:
Correct